racingcar-java/src/main/java/Calculator.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Objects;
+
+public class Calculator {
+
+    private static final String BLANK = " ";
+    private static final String PLUS = "+";
+    private static final String MINUS = "-";
+    private static final String MULTIPLY = "*";
+    private static final String DIVIDE = "/";
+
+    int calculate(String input) {
+        checkNullOrEmpty(input);
+
+        String[] data = input.split(BLANK);
+        checkInputLength(data);
+
+        checkValidNumber(data[0]);
+        int result = Integer.parseInt(data[0]);
+
+        for (int i = 1; i < data.length; i += 2) {
+            checkValidOperator(data[i]);
+            String operator = data[i];
+
+            checkValidNumber(data[i+1]);
+            int num = Integer.parseInt(data[i+1]);
+
+            result = operate(result, operator, num);
+
+        }
+        return result;
+    }
+
+    private int operate(int result, String operator, int num) {
+        if (operator.equals(PLUS)) {
+            result = plus(result, num);
+        } else if (operator.equals(MULTIPLY)) {
+            result = multiply(result, num);
+        } else if (operator.equals(MINUS)) {
+            result = minus(result, num);
+        } else if (operator.equals(DIVIDE)) {
+            result = divide(result, num);
+        }
+        return result;
+    }
```

Comment:
```suggestion
    private int operate(int result, String operator, int num) {
        if (operator.equals(PLUS)) {
            return plus(result, num);
        } else if (operator.equals(MULTIPLY)) {
            return multiply(result, num);
        } else if (operator.equals(MINUS)) {
            return minus(result, num);
        } else if (operator.equals(DIVIDE)) {
            return divide(result, num);
        }
        throw new RuntimeException("더이상 사칙연산이 존재하지 않습니다");
    }
```
변경처럼 바로 리턴을 하고, 어느 operator에 속하지않는 경우 exception이 발생하도록 진행하는게 로직상 나아보일수도 있을 것 같네요 :)